sherlock_project/resources/data.json

```diff
@@ -2777,6 +2784,13 @@
     "urlMain": "https://www.toster.ru/",
     "username_claimed": "adam"
   },
+  "udemy": {
+    "errorMsg": "Page not found",
+    "errorType": "message",
+    "url":"https://www.udemy.com/user/{}",
+    "urlMain": "https://www.udemy.com/",
+    "username_claimed": "adfadf"
+  },
```

Comment:
Returns positive for all queries
`errorType` of `message` should be a last resort behind status_code due to l10n issues